### PR TITLE
Demo/stream api

### PR DESCRIPTION
**Pull request by sogunshola**
---
name: 'Pull request'
about: A new pull request
---

### Checklist

- [X] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [X] My code is conform the [code style](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/CODE_STYLE.md)
- [X] I have made corresponding changes to the documentation
- [X] I have updated Typescript definitions when needed

### Issue Description

Related issue: #`FILL_THIS_OUT`

### CODE REVIEW

File: demos/moralis-stream/src/hook.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  });
+}
+
+function verifySignature(req: Request) {
+  const providedSignature = req.headers['x-signature'];
+  if (!providedSignature) {
+    throw new Error('No signature provided');
+  }
+
+  const generatedSignature = Web3.utils.sha3(JSON.stringify(req.body) + config.STREAM_SECRET);
```
Comment (b4rtaz) [inline demos/moralis-stream/src/hook.ts:46]: You use only `sha3` method form the `web3` package. Maybe would be better if you would use specific package ([sha3](https://www.npmjs.com/package/sha3))?

File: demos/moralis-stream/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+1. Copy `.env.example` to `.env` and fill in the values
+2. Run `yarn install`
+3. Run `yarn dev` to run the server locally
+
+Now your server is running locally and also an ngrok tunnel has been started and logged in your console with the following endpoints:
+
+- Client: `localhost:3000` (or any other port you set in `.env`)
+- API: `localhost:3000/stream` (or any other port you set in `.env`)
+- Webhook: `{ngrokurl}/hooks/stream`
```
Comment (b4rtaz) [inline demos/moralis-stream/README.md:13]: Here should be info how to configure ngrok (what is it, how to create an account etc).

File: demos/moralis-stream/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "@moralisweb3/eslint-config": "^1.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.12.0",
+    "@typescript-eslint/parser": "^5.12.0",
+    "eslint": "^8.9.0",
+    "eslint-plugin-import": "^2.26.0",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.7.4"
+  },
+  "scripts": {
+    "dev": "ts-node src/index.ts",
```
Comment (b4rtaz) [inline demos/moralis-stream/package.json:26]: Rename `dev` into `start` or add `"start": "npm run dev"`.

File: demos/moralis-stream/public/styles.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
+html {
+  font-family: 'Roboto', sans-serif;
```
Comment (b4rtaz) [inline demos/moralis-stream/public/styles.css:2]: `'Roboto', sans-serif;` => `Roboto, Arial, Verdana, Tahoma, sans-serif;`

File: demos/moralis-stream/src/stream/streamService.ts [hunk 5]
```diff
@@ -0,0 +1,78 @@
+import Moralis from 'moralis';
+
+interface StreamOptions {
+  network: 'evm';
+  webhookUrl: string;
+}
+
+const DESCRIPTION = 'some description';
+const TAG = 'transactions';
+const ADDRESS = '0x992eCcC191D6F74E8Be187ed6B6AC196b08314f7';
```
Comment (b4rtaz) [inline demos/moralis-stream/src/stream/streamService.ts:10]: Should we hardcode it? Maybe it should be in env variables?

**Review by b4rtaz — CHANGES_REQUESTED** [review]

File: demos/moralis-stream/README.md (hunk 2 above)
Comment (sogunshola) [reply demos/moralis-stream/README.md:13]: Okay. I'll add a link there, but right now, nothing needs to be done, just `yarn install`

**Review by b4rtaz — APPROVED** [review]

File: demos/moralis-stream/src/hook.ts (hunk 1 above)
Comment (b4rtaz) [reply demos/moralis-stream/src/hook.ts]: 👍

**Pull request merged**